src/concrete/ml/common/utils.py

```diff
@@ -583,3 +582,33 @@ def all_values_are_of_dtype(*values: Any, dtypes: Union[str, List[str]]) -> bool
         supported_dtypes[dtype] = supported_dtype
 
     return all(_is_of_dtype(value, supported_dtypes) for value in values)
+
+
+def check_isclose_and_shape(a, b, **kwargs):
```

Comment:
better to put the few parameters handled by isclose instead of kwargs imo, it's less error-prone !